lstchain/io/dataselection.py

```diff
@@ -0,0 +1,263 @@
+from ctapipe.core import Component
+from ctapipe.core.traits import Int, Float, List
+import numpy as np
+import astropy.units as u
+from pyirf.binning import create_bins_per_decade  # , add_overflow_bins
+
+__all__ = ["DataSelection", "DataBinning"]
+
+
+class DataSelection(Component):
```

Comment:
Can you review this and the test for these Componenets @maxnoe? I think, I have written it in a minimalistic way, as the use of the Components are not so intrusive with other scripts/tools.

Reply:
Something like this:

```
class EventFilter(Component):
    filters = Dict().tag(config=True)
    finite_parameters = List().tag(config=True)

    def __call__(self, events):
         return filter_events(events, self.filters, self.finite_parameters)
```

Should be enough right?